### PR TITLE
Added the ideas generated from our last meeting.

### LINKED ISSUE #3: Generate more new ideas
Related to https://github.com/UBC-LA-Hackathon/student-dashboard/issues/2.

I came up with many feature ideas for heatmap, but not that many for completely unrelated tools/visualizations. 

We should add at least 3 more intermediate and 3 more advanced. 

Should we separate out the categories into beginner, intermediate and advanced?

### PR DESCRIPTION
**Pull request by justin0022**
Resolves #3

### CODE REVIEW

File: 5-Create-Advanced-Data-Viz.md [hunk 1]
```diff
@@ -41,12 +43,17 @@ Don't want to build new features for the heatmap? That's okay!
 TODO: Any more data?
 
 #### Intermediate
-* Create a degree planning tool that shows students what courses they are eligible to register for, given a list of courses in Canvas students have completed and the [degree requirments data (CSV)](TODO: LINK)
-*
+* Create a degree planning tool that shows students what courses they are eligible to register for, given a list of courses in Canvas students have completed and the [degree requirements data (CSV)](TODO: LINK)
```
Comment (markoprodanovic) [inline 5-Create-Advanced-Data-Viz.md]: nit: add period after (TODO: LINK)

File: 5-Create-Advanced-Data-Viz.md [hunk 2]
```diff
@@ -41,12 +43,17 @@ Don't want to build new features for the heatmap? That's okay!
...
+* Create a tool that allows you to compare courses that share the same information from a selected field using [course calendar data (CSV)](TODO: Link)
```
Comment (markoprodanovic) [inline 5-Create-Advanced-Data-Viz.md]: nit: add period after (TODO: LINK)

**Comment by justin0022** [comment]
Should be fixed now.

**Review by markoprodanovic — COMMENTED** [review]
Looks good!

**Comment by justin0022** [comment]
> Looks good!

Could you approve the PR?

**Comment by markoprodanovic** [comment]
Yup! Just gotta figure out how to fix merge conflicts now that the file's been renamed in master.

**Comment by justin0022** [comment]
Oh just fixed it!

**Pull request merged**